stack/__main__.py

```diff
@@ -650,6 +657,117 @@ def add_access_group_cache_as_secret_member(secret, resource_prefix: str):
             member=pulumi.Output.concat('group:', release_access_group.group_key.id),
         )
 
+    if enable_egress_project:
+        if not release_bucket:
+            raise ValueError(
+                'Requested egress project, but no bucket is available to egress from'
+            )
+
+        egress_buckets = {'release': release_bucket}
+
+        project_name = f'cpg-{dataset}-egress'
```

Comment:
How about we make this a bit more generic, so we can potentially use this for compute as well in the future? Can we make the name consistent with the existing project names?

```suggestion
        project_name = f'{project_id}-shared'
```

Reply:
Do you want to rename all the other "egress" occurrences as well?

Reply:
Ah right, I see sorry.

Reply:
Tested and pushed new changes :)